Tribute/Tribute.swift

```diff
@@ -119,6 +121,17 @@ private extension Attributes.GlyphDirection {
 }
 
 extension Attributes {
+    
+    /// convenience method for comparing attributes on `paragraph` vs `defaultParagrah`
+    private func paraStyleCompare< U:Equatable>(trans:NSParagraphStyle->U) -> U?{
```

Comment:
move it after initializer please, 
also if you don't mind changing spacing to:
```swift
private func paraStyleCompare<U:Equatable>(paragraph: NSParagraphStyle, transformation: NSParagraphStyle -> U) -> U? {
```
 it's a silly thing, but I prefer to keep code style similar across the project
Also I included the paragraph as a parameter to avoid keeping it as a property on the structure